### ISSUE TITLE
[CAGradientLayer Extension] Error: Argument passed to call that takes no arguments

### ISSUE THREAD

**Issue by muxinqi** [SwifterSwift/SwifterSwift#999]
**Describe the bug**
After import `SwifterSwift`, when I try to use the convenience initiate method of `CAGradientLayer`, Xcode shows the following error:
```error
Argument passed to call that takes no arguments
Remove '(
    colors: [Color.red, Color.blue],
    startPoint: CGPoint(x: 1, y: 0.5),
    endPoint: CGPoint(x: 0.5, y: 0.5)
)'
```

**To Reproduce**
1. Create a minimal iOS project
2. Import `SwifterSwift`
3. Try to init a `CAGradientLayer` with [the convenience method](https://github.com/SwifterSwift/SwifterSwift/blob/dc813f3aa92f904a009fe543b99d5649dce7c134/Sources/SwifterSwift/CoreAnimation/CAGradientLayerExtensions.swift#L25)

Code:
```swift
import UIKit
import SwifterSwift

class ViewController: UIViewController {

    let layer = CAGradientLayer( // Compiler error will prompt here
        colors: [UIColor.red, UIColor.blue],
        startPoint: CGPoint(x: 1, y: 0.5),
        endPoint: CGPoint(x: 0.5, y: 0.5)
    )
    
    override func viewDidLoad() {
        super.viewDidLoad()
        // Do any additional setup after loading the view.
    }

}
```

**Expected behavior**
No error prompt.

**Screenshots**
<img width="721" alt="image" src="https://user-images.githubusercontent.com/42941327/147625369-2bd0a6b9-f2a2-4aa4-8070-c92a67c57869.png">

**Additional context**
 - macOS Monterey 12.1
 - Xcode 13.2.1
 - SwifterSwift 5.2.0

- [x] I've read, understood, and done my best to follow the [Contributing guidelines](https://github.com/SwifterSwift/SwifterSwift/tree/master/CONTRIBUTING.md) before opening this issue.

**Comment by TTOzzi** [comment]
This problem has already been solved at https://github.com/SwifterSwift/SwifterSwift/pull/856.
When https://github.com/SwifterSwift/SwifterSwift/issues/996 is solved, it seems to be solved together naturally.
You can solve it temporarily using `pod 'SwifterSwift', :git => 'https://github.com/SwifterSwift/SwifterSwift.git', :branch => 'master'` at Podfile.

**Comment by muxinqi** [comment]
> This problem has already been solved at #856. When #996 is solved, it seems to be solved together naturally. You can solve it temporarily using `pod 'SwifterSwift', :git => 'https://github.com/SwifterSwift/SwifterSwift.git', :branch => 'master'` at Podfile.

Pulled the latest code and tried again, as you said, the problem has been solved.
Thank you for your efforts!
